src/pages/_document.tsx

```diff
@@ -37,9 +37,14 @@ class MyDocument extends Document {
     return (
       <Html>
         <Head>
-          <link rel="preconnect" href="https://fonts.gstatic.com" />
+          <link rel="preconnect" href="https://fonts.googleapis.com" />
           <link
-            href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,600;0,700;1,400&display=swap"
+            rel="preconnect"
+            href="https://fonts.gstatic.com"
+            crossOrigin=""
```

Comment:
Either make it `crossOrigin="anonymous"` or remove it at all

More details: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin

Reply:
Literally copy this from google fonts.

Reply:
empty and anonymous are the same, but is omitting attr `crossorigin` the same though?

Reply:
Yea but it just looks bad with empty value like that... Even Google doesn't follow coding standard I guess